binary_file.py

```diff
@@ -0,0 +1,5 @@
+from file import File
+
+
+class BinaryFile(File):
```

Comment:
Where is all of the documentation?

Reply:
Look at the documentation section, pick a format of your liking and start documenting every function\class